src/com/latacora/backsaws/ini.clj

```diff
@@ -0,0 +1,40 @@
+(ns com.latacora.backsaws.ini
+  (:require
+   [clojure.java.io :as io]
+   [instaparse.core :as insta]
+   [babashka.fs :as fs]
+   [meander.epsilon :as m]))
+
+(def ini-parser (-> "ini.ebnf" io/resource insta/parser))
+(defn parse-path! [path] (-> path fs/expand-home fs/file slurp ini-parser))
+(def parse-aws-config! (partial parse-path! "~/.aws/config"))
+
+(defn ^:private get-sectionless-kvs
+  [ini-parse]
```

Comment:
nbd, minor maintainability suggestion, this arg might be easier to understand for future maintainers were it named `parsed` (the past tense is the ~important part, but I also think we can drop the `ini` given the context) (and this’d apply to the following two functions as well) (and for the public function, it’d also be a minor API improvement)

Reply:
Done!

Reply:
(I've commonly seen `parse` as a noun, which is why it didn't read that way to me, but I agree this is better)